openff/toolkit/utils/rdkit_wrapper.py

```diff
@@ -491,8 +875,91 @@ def _get_connectivity_from_openmm_top(self, omm_top):
 
         return rwmol
 
+    def _get_symmetrical_groups(self, fuzzy_query, substruct):
```

Comment:
This bit of code was added as a way to handle some tricky edge cases. For example, take carboxylate ions ,-C(-[O-])(=O). The code later one might error because the chemical information changes based on whether the graph match places the negative charge/double bond on one oxygen or another. 
In other cases of highly symmetric groups (including lots of aromatic cases) this code helps catch what would be chemical ambiguity errors. I'll add some comments and tests cases for this.